.github/workflows/pylint.yaml

```diff
@@ -0,0 +1,18 @@
+name: Pylint
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: python-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v3
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
+    - name: Analysing the code with pylint
+      run: |
+        pylint $(git ls-files '*.py')
```

Comment:
is it possible to use the integrated linter from the project ? there is ruff, black and mypy, why using pylint here ? We should be sync from our dev env and the CI env right ? 🤔

Reply:
😯 , i hadn't forgotten it.

Reply:
https://github.com/dylantientcheu/lamap-bot/blob/v2/Makefile#L18

Reply:
I updated